_docs/cardano/2017-01-06-updating.md

```diff
@@ -110,28 +110,25 @@ new version blocks are still compatible with old version clients.
 A hard fork is one that doesn’t maintain backward compatibility with
 the previous version.
 
-BIP-99 provides an excellent criteria to distinguish between these two
+BIP-99 provides excellent criteria to distinguish between these two
 types of fork:
 
-### Soft fork
+* A **soft fork** is a consensus fork wherein everything that was previously
+  invalid remains invalid while blocks that would have been previously considered
```

Comment:
Be careful with your quantification - not all blocks that were previously valid become invalid. Only "some blocks", although you don't need to go on to explain why. Maybe just explain that a soft-fork introuces new rules that cause this.